encoding/csv_test.ts

```diff
@@ -473,16 +473,13 @@ for (const t of testCases) {
       let actual;
       if (t.Error) {
         const err = await assertThrowsAsync(async () => {
-          await readMatrix(
-            new BufReader(new StringReader(t.Input ?? "")),
-            {
-              separator,
-              comment: comment,
-              trimLeadingSpace: trim,
-              fieldsPerRecord: fieldsPerRec,
-              lazyQuotes: lazyquote,
-            },
-          );
+          await readMatrix(new BufReader(new StringReader(t.Input ?? "")), {
```

Comment:
We don't need the formatting change, you can revert this

Reply:
@wperron fixed

Reply:
Thanks, but I'm really just interested in the changes on line 452 and 624

Reply:
@wperron done